src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/Content/PageBody/_details.html.twig

```diff
@@ -207,3 +209,130 @@
         </table>
     </div>
 </div>
+{% else %}
+{% set variant = product.variants.first %}
+<div class="card mb-3">
+    <div class="card-header">
+        <div class="card-title">
+            {{ 'sylius.ui.details'|trans }}
+        </div>
+    </div>
+    <table>
+        <tr class="bg-gray-400">
+            <td colspan="9" class="p-0 border-0">
+                <div class="py-3 px-5" id="product-variant-details-{{ variant.code }}">
+                    <div class="card mb-3">
+                        <div class="card-header">
+                            <div class="card-title">{{ 'sylius.ui.pricing'|trans }}</div>
+                        </div>
+                        <div class="table-responsive">
```

Comment:
Guessing this whole table could be extracted into a separate template and reused both here and within the configurable part, inserted via a hook preferably.